apps/mobile/src/queries/market-data/market-data.query.ts

```diff
@@ -0,0 +1,46 @@
+import { useSettings } from '@/store/settings/settings';
+import { QueryFunctionContext, useQueries, useQuery } from '@tanstack/react-query';
+
+import { bitcoinNetworkModeToCoreNetworkMode } from '@leather.io/bitcoin';
+import { FungibleCryptoAssetInfo, NetworkModes } from '@leather.io/models';
+import { getMarketDataService } from '@leather.io/services';
+import { oneMinInMs } from '@leather.io/utils';
+
+export function createMarketDataQueryOptions(
+  token: FungibleCryptoAssetInfo,
+  network: NetworkModes
+) {
+  return {
+    queryKey: ['market-data-service-get-market-data', network, token],
```

Comment:
It doesn't need to be for this PR but sometimes we use `enum`s for the keys and sometimes not. e.g. `BitcoinQueryPrefixes` .

We should probably standardise that